dive_core/command_hierarchy.cpp

```diff
@@ -1220,8 +1220,7 @@ uint64_t CommandHierarchyCreator::AddPacketNode(const IMemoryManager &mem_manage
                             packet_node_index);
         }
         else if (type7_header.opcode == Pal::Gfx9::IT_LOAD_UCONFIG_REG ||
-                type7_header.opcode == Pal::Gfx9::IT_LOAD_CONTEXT_REG ||
```

Comment:
is this part of this change? i guess i could do a full path to remove all these gfx9 commented code?

Reply:
Oh crap did I remove that? Not intentional

Reply:
Reverted this accidental change